src/gordon_gcp/plugins/janitor/authority.py

```diff
@@ -141,13 +141,11 @@ def __init__(self, config, crm_client, gce_client, rrset_channel=None,
         # project - blacklist.
         return sorted(projects - project_blacklist)
 
-    def _filter_results(self, projects, results):
+    def _filter_results(self, results):
         successful_results = []
         for index, result in enumerate(results):
             if isinstance(result, exceptions.GCPHTTPError):
-                project = projects[index]
-                msg = (f'Could not fetch instance list for project {project}: '
-                       f'{result}. Skipping this project.')
```

Comment:
I think we should keep the 'skipping' bit.

Reply:
Hmm, ok